tests/test_smart_ptr.cpp

```diff
@@ -101,7 +101,7 @@ class MyObject3 : public std::enable_shared_from_this<MyObject3> {
 // test_unique_nodelete
 // Object with a private destructor
 class MyObject4;
-static std::unordered_set<MyObject4 *> myobject4_instances;
+std::unordered_set<MyObject4 *> myobject4_instances;
```

Comment:
These static keywords are redundant in an anonymous namespace

Reply:
Awesome to have a check for this (I can never remember all the arcane rules).